models/monai_prostate158/utils/Prostate158Runner.py

```diff
@@ -0,0 +1,85 @@
+"""
+-------------------------------------------------
+MHub - MONAI Prostate158 Runner
+-------------------------------------------------
+
+-------------------------------------------------
+Author: Cosmin Ciausu
+Email:  cciausu97@gmail.com
+-------------------------------------------------
+"""
+# TODO: support multi-i/o and batch processing on multiple instances
+
+from typing import List, Optional
+import os, subprocess, shutil, glob, sys
+import SimpleITK as sitk, numpy as np
+from mhubio.core import Module, Instance, InstanceData, DataType, FileType, IO
+from mhubio.modules.runner.ModelRunner import ModelRunner
+import json
+
+@IO.Config('apply_center_crop', bool, False, the='flag to apply center cropping to input_image')
+
+class Prostate158Runner(ModelRunner):
+
+    apply_center_crop : bool
+
+    @IO.Instance()
+    @IO.Input("in_data", 'nifti:mod=mr', the="input T2 sequence data to run prostate158 on")
+    @IO.Output('out_data', 'monai_prostate158.nii.gz', 
+               'nifti:mod=seg:model=Prostate158:roi=PROSTATE_TRANSITION_ZONE,PROSTATE_PERIPHERAL_ZONE', 
+               data='in_data', bundle='model', the="predicted segmentation model")
+
+    def task(self, instance: Instance, in_data: InstanceData, out_data: InstanceData) -> None:
+
+        # bring input data in nnunet specific format
+        # NOTE: only for nifti data as we hardcode the nnunet-formatted-filename (and extension) for now.
+        assert in_data.type.ftype == FileType.NIFTI
```

Comment:
These don't do any harm, however, as you explicitly request `nifti`from the `@IO.Input` decorator you can be sure to only get files of type NIFTI here. The extension could be different though (e.g. uncompressed nifti).